vspace/src/main/webapp/WEB-INF/views/staff/user/list.html

```diff
@@ -1,47 +1,47 @@
-		        
-		        </div>
-		    </div>
-	</div>
+<html layout:decorate="~{layouts/main_staff}">
+<head>
+</head>
+
+<body>
+	<div layout:fragment="content" class="main-content">
+		<h2>Users</h2>
+        <div  th:each="user: ${users}" class="tab" style="display: flex; justify-content: space-between">
+		    	<h5 th:if="${#lists.contains(user.roles, 'STAFF')}" class="badge badge-primary">Staff</h5>
+		        [[${user.username}]] ([[${user.firstName}]] [[${user.lastName}]])
+		        <div>
+		        	<span th:if="${!user.enabled}" class="badge badge-light">Account Deactivated</span>
+			        <span th:if="${user.admin}" class="badge badge-info" style="background: var(--primary)">Admin User</span>
+			        <span th:if="${user.enabled}" class="badge badge-success">Active</span>
+		        </div>
+		        
+		        <div>
+		        
+	            <form th:if="${!user.enabled}" th:action="@{'/staff/user/'+${user.username}+'/approve'}"  th:object="${approveUrl}" method="POST">
+	            	<button style="padding:0" class="btn btn-link"><span><i class="fas fa-user-check"></i></span> Activate</button>
+	            </form>
+		        <div th:if="${user.enabled}" style="display: flex">
+                    <form th:if="${!user.admin}" th:action="@{'/staff/user/'+${user.username}+'/admin'}" th:object="${postUrl}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-key"></i></span> Make Admin</button>
+                    </form>
+                    <form th:if="${user.admin}" th:action="@{'/staff/user/'+${user.username}+'/admin/remove'}" th:obejct="${removeUrl}" method="POST">
+                    <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user"></i></span> Remove Admin Role</button>
+                    </form>
+                    
+                    <form th:if="${!#lists.contains(user.roles, 'STAFF')}" th:action="@{'/staff/user/'+${user.username}+'/role/add?roles=ROLE_STAFF'}" th:object="${url}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user"></i></span> Make Staff</button>
+                    </form>
+                    
+                    <form th:if="${#lists.contains(user.roles, 'STAFF')}" th:action="@{'/staff/user/'+${user.username}+'/role/remove?roles=ROLE_STAFF'}" th:object="${url}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user"></i></span> Remove Staff Role</button>
+                    </form> 
+                
+                    <form th:action="@{'/staff/user/'+${user.username}+'/disable'}" th:object="${disableUrl}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user-slash"></i></span> Deactivate User</button>
+                    </form>
+                </div>
+		        
+		        </div>
+        </div>
+    </div>
```

Comment:
Still the same question, why is this file being changed?

Reply:
I am not sure, why this file is being changed. I have replaced the current version with the version of `VSPC-120` ticket.

Reply:
It's still here

Reply:
This file is the version from `VSPC-120` ticket. Do I have to change the line endings? or do a File -> format?

Reply:
If this were the file from 120, it wouldn't show up as changed.

Reply:
I replaced the contents of the file from 120 to 89. Which is why I had to commit and push them. 
What should I do with this file?

Reply:
revert it if you didn't change anything. Whatever you did, you changed the file, otherwise it wouldn't show up as changed in the pr changes tab.

Reply:
Okay. I did File -> Replace With -> Branch -> Develop version

Reply:
You pr is not to develop though.

Reply:
Should I replace it with `HEAD REVISION` ?

Reply:
It should be the version of the branch your are making the pr to

Reply:
So I am making the PR to vspc-120. Then I should be replacing the file to branch 120 version right?

Reply:
Yes, replace the file with the one from 120.

Reply:
Yes, I replaced the one from VSPC-120